api/src/main/java/javax/mvc/binding/BindingResult.java

```diff
@@ -99,31 +99,31 @@
     BindingError getBindingError(String param);
 
     /**
-     * Returns an immutable set of all constraint violations detected.
+     * Returns an immutable set of all validation errors detected.
      *
-     * @return All constraint violations detected
+     * @return All validation errors detected
      */
-    Set<ConstraintViolation<?>> getAllViolations();
+    Set<ValidationError> getAllValidationErrors();
```

Comment:
Hey Christian,

Yes, I totally agree that the spec contracts should be consistent. 
+1 for keep as it is and revisit this later